reolinkapi/mixins/network.py

```diff
@@ -72,6 +72,27 @@ def set_wifi(self, ssid: str, password: str) -> Dict:
             }}}]
         return self._execute_command('SetWifi', body)
 
+    def set_ntp(self, enable: bool = True, interval: int = 1440, port: int = 123, server: str = "pool.ntp.org") -> Dict:
+        """
+        Set NTP settings.
+
+        :param enable: bool
+        :param interval: int
+        :param port: int
+        :param server: str
+        :return: Dict
+        """
+        body = [{"cmd": "SetNtp", "action": 0, "param": {
+            "Ntp": {
+                "enable": 1 if enable else 0,
```

Comment:
can't you just have `enable` as the value? The bool translates to a 1 or 0 anyways in JSON, right?

Reply:
I get the following error when trying to use the strait boolean.

```
[{'cmd': 'SetNtp', 'code': 1, 'error': {'detail': 'param error', 'rspCode': -4}}]
```

Reply:
I actually just tried `set_osd` and got the same error. It only worked when I edited the function to use the same logic I have here.

Reply:
right, it seems the behavior of Python is to convert to the JSON `false/true` literal. My bad, in Go it gets converted to a primitive (int) so either 1 or 0. 

Then I think we should force it here to an in like so
```suggestion
                "enable": int(enable),
```